packages/data/addon/serializers/facts/elide.ts

```diff
@@ -8,19 +8,43 @@
 import EmberObject from '@ember/object';
 import NaviFactSerializer, { ResponseV1 } from './interface';
 import { AsyncQueryResponse, RequestV2 } from 'navi-data/adapters/facts/interface';
+import { getElideField } from 'navi-data/adapters/facts/elide';
+import { canonicalizeMetric } from 'navi-data/utils/metric';
 
 export default class ElideFactsSerializer extends EmberObject implements NaviFactSerializer {
+  /**
+   * @param payload - raw payload string
+   * @param request - request object
+   */
+  private processResponse(payload: string, request: RequestV2): ResponseV1 {
+    const response = JSON.parse(payload);
+
+    const { table } = request;
+    const elideFields = request.columns.map(({ field, parameters }) => getElideField(field, parameters));
+    const normalizedFields = request.columns.map(({ field: metric, parameters }) =>
+      // TODO rename with generic canonicalizeColumn
+      canonicalizeMetric({ metric, parameters })
+    );
+
+    const rawRows = response.data[table].edges;
+    const totalRows = rawRows.length;
+    const totalFields = normalizedFields.length;
+    const rows = new Array(totalRows);
+
+    let r = totalRows;
+    while (r--) {
+      const rawRow = rawRows[r].node;
+      const newRow: Record<string, unknown> = (rows[r] = {});
+      let f = totalFields;
+      while (f--) {
+        newRow[normalizedFields[f]] = rawRow[elideFields[f]];
```

Comment:
We'll need to confirm that the fields in the response will have the parameters added to them in the same format. Can we add a TODO to revisit this once Elide has parameterized column support?

Reply:
Also, do we have a test with a parameterized column?

Reply:
We need to revisit elide parameters. We may need to implement aliases. Currently, the mock does not response with the params and requesting the same field with two different param values returns nothing. Since elide does not support params at the moment, we can skip the test.